code/modules/jobs/job_types/roguetown/garrison/mayor.dm

```diff
@@ -65,4 +64,32 @@
 		H.change_stat("speed", -3)
 		H.change_stat("intelligence", 5)
 		ADD_TRAIT(H, TRAIT_WANTED_POSTER_READ, TRAIT_GENERIC)
+		ADD_TRAIT(H, TRAIT_BOGVULNERABLE, TRAIT_GENERIC)
```

Comment:
Don't particularly think the mayor needs the bog vulnerable trait as they're not a 'combat' role persay, but it's not bad for them to have either.

Reply:
i probably will leave this here since i don't want mayors to go to the bog.